code/game/mecha/medical/odysseus.dm

```diff
@@ -44,3 +49,29 @@
 //TODO - Check documentation for client.eye and client.perspective...
 /obj/item/clothing/glasses/hud/health/mech
 	name = "Integrated Medical Hud"
+
+
+
+//A adminspawn only syndicate variant with slightly more armor, health and a pre-loaded laser gun. Yeah this is the most uncreative shit ever but it's a starting point if someone wants to expand on this.
+/obj/mecha/medical/odysseus/murdysseus
+	name = "MURDYSSEUS"
+	desc = "A terrifying combat-modified Medical Exosuit. You doubt this thing has ever heard of the Hippocratic Oath."
+	icon_state = "murdysseus"
+	initial_icon = "murdysseus"
+	deflect_chance = 20
+	health = 240
+	wreckage = /obj/effect/decal/mecha_wreckage/odysseus/murdysseus
+	paintable = 0
+	max_equip = 4
+
+/obj/mecha/medical/odysseus/murdysseus/New()
+	..()
+	var/obj/item/mecha_parts/mecha_equipment/ME = new /obj/item/mecha_parts/mecha_equipment/weapon/energy/laser
+	ME.attach(src)
```

Comment:
honestly the New() of mecha_equipment should automatically check if it's inside a mecha and attach if so - why not do that instead and futureproof us?

Reply:
how do I check if the item is inside a mech? give New a parameter?

Reply:
not a param, so what you would do is on line 69 here add (src) to the top, which means its spawn loc is the mech

then in mecha_equipment/New() add this
```
if(istype(loc,/obj/mecha))
  attach(loc)
```

Reply:
You know if you don't mind I'll do this in another PR. I'm not confident enough with code to figure this out on my own.

Reply:
your comment went back in time, strange. anyway I did what you said and this is now resolved, tested and working .